client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/Sections/Details.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { ReactElement } from 'react';
+import {
+  ReasonOptionsSearchInput,
+  RequestLineFragment,
+} from '@openmsupply-client/system';
+import {
+  Grid,
+  InputWithLabelRow,
+  NumUtils,
+  Plugins,
+  ReadOnlyInput,
+  ReasonOptionNodeType,
+  Stack,
+  useAuthContext,
+  useFormatNumber,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useRequest } from '../../../api';
+import { DraftRequestLine } from '../hooks';
+import { RequestStats } from '../ItemCharts/RequestStats';
+
+const LABEL_WIDTH = '400px';
+
+interface DetailsProps {
+  update: (patch: Partial<DraftRequestLine>) => void;
+  plugins: Plugins;
+  draft?: DraftRequestLine | null;
+  save?: () => void;
+  isProgram?: boolean;
+  isPacksEnabled?: boolean;
+  disabled?: boolean;
+  line?: RequestLineFragment;
+}
+
+export const Details = ({
+  draft,
+  save,
+  update,
+  isProgram,
+  isPacksEnabled,
+  plugins,
+  disabled,
+  line,
+}: DetailsProps): ReactElement => {
+  const t = useTranslation();
+  const { store } = useAuthContext();
+  const formatNumber = useFormatNumber();
+
+  const { maxMonthsOfStock, minMonthsOfStock } = useRequest.document.fields([
+    'maxMonthsOfStock',
+    'minMonthsOfStock',
+  ]);
+
+  const targetStock =
+    (draft?.itemStats.averageMonthlyConsumption ?? 0) * maxMonthsOfStock;
+
+  const useConsumptionData =
+    store?.preferences?.useConsumptionAndStockFromCustomersForInternalOrders;
+
+  const NumericReadOnlyInput = ({ value }: { value?: number | null }) => {
+    return (
+      <ReadOnlyInput value={NumUtils.round(value ?? 0, 2).toString()} number />
+    );
+  };
+
+  return (
+    <>
+      <Grid container spacing={10} direction="row">
+        {/* Left Column Content */}
+        <Grid size={{ xs: 12, sm: 6 }}>
+          <Stack spacing={1.5}>
+            <InputWithLabelRow
+              Input={
+                <ReadOnlyInput
+                  value={draft?.unitName ?? ''}
+                  style={{ textAlign: 'right' }}
+                />
+              }
+              labelWidth={LABEL_WIDTH}
+              label={t('label.unit')}
+            />
```

Comment:
This block is pretty heavily repeated... want to create a reusable component, just pass in the label and value?